runtime-modules/governance/src/council.rs

```diff
@@ -75,6 +85,73 @@ impl<T: Trait> Module<T> {
         CouncilMint::<T>::put(mint_id);
         Ok(mint_id)
     }
+
+    fn add_reward_relationship(
+        destination: &T::AccountId,
+    ) -> Result<T::RewardRelationshipId, recurringrewards::RewardsError> {
+        if let Some(reward_source) = Self::council_mint() {
+            let recipient = <recurringrewards::Module<T>>::add_recipient();
+
+            let next_payout_at = system::Module::<T>::block_number()
+                + Self::first_payout_after_term_start()
+                + T::BlockNumber::one();
+
+            let relationship_id = <recurringrewards::Module<T>>::add_reward_relationship(
+                reward_source,
+                recipient,
+                destination.clone(),
+                Self::amount_per_payout(),
+                next_payout_at,
+                Self::payout_interval(),
+            )?;
+
+            RewardRelationships::<T>::insert(destination, relationship_id);
+
+            Ok(relationship_id)
+        } else {
+            Err(recurringrewards::RewardsError::RewardSourceNotFound)
+        }
+    }
+
+    fn remove_reward_relationships() {
+        for seat in Self::active_council().into_iter() {
+            if RewardRelationships::<T>::exists(&seat.member) {
+                let id = Self::reward_relationships(&seat.member);
+                <recurringrewards::Module<T>>::remove_reward_relationship(id);
+            }
+        }
+    }
+
+    fn apply_council_seats(seats: Seats<T::AccountId, BalanceOf<T>>, term: T::BlockNumber) {
```

Comment:
What does this method actually do? this name doesn't say muhc, and is hard to reconcile with the body.

Reply:
this just has the body of code from `council_elected()` I moved it into its own method because I was having issue calling it from tests but later realized it was just a scoping issue.
I can either rename it to `on_council_elected()` or just move the body back into it original location.

Reply:
Was moved back.